utils/install.sh

```diff
@@ -576,12 +576,10 @@ main() {
 
     local _source=". \"$IPATH/env\""
     local _grep=$(cat "$__HOME__/.profile" 2>/dev/null | grep "$IPATH/env")
-    if [ -f "$__HOME__/.profile" ]; then
-        if [ "$_grep" = "" ]; then
-            echo "$_source" >>"$__HOME__/.profile"
+    if [ "$_grep" = "" ]; then
+        if [ ! -f "$__HOME__/.profile" ]; then
+            echo "Generating $__HOME__/.profile"
         fi
-    else
-        echo "Generating $__HOME__/.profile"
         echo "$_source" >>"$__HOME__/.profile"
     fi
```

Comment:
I think this refactoring does the same job but it's more readable.

Reply:
> I think this refactoring does the same job but it's more readable.

Yes, it's shorter.